src/runtime/analytics-service.js

```diff
@@ -155,6 +155,13 @@ export class AnalyticsService {
     this.context_.setSku(sku);
   }
 
+  /**
+   * @param {string} url
+   */
+  setUrl(url) {
+    this.context_.setUrl(url);
```

Comment:
Could you give this method test coverage?

Reply:
Ok